### DISCUSSION TITLE
Separate Breaking Changes from Non-Breaking changes in the ouput support

### DISCUSSION THREAD

**Discussion by WillGunn** [Tufin/oasdiff#134] [Ideas]
If the report output has a mix of breaking and non-breaking changes, it would be useful to be able to have those differences grouped separately by their breaking status.  It could also be resolved by having a -non-breaking-only flag so you can run the tool twice to generate a list of breaking changes and a list of non breaking changes.

**Comment by reuvenharrison** [comment]
Hi Will,
Can you please explain the use case?
How do you intend to use the non-breaking changes?
Thanks,
Reuven

**Reply by WillGunn** [reply]
@reuvenharrison  We are evaluating tools to generate an automated api contract change log and would like to be able to separate which changes are breaking from non-breaking (as evaluated by the tool).

**Reply by reuvenharrison** [reply]
I am researching a potential solution. Stay tuned.

**Reply by WillGunn** [reply]
> I am researching a potential solution. Stay tuned.

Just checking in to see if you made any headway in this. @reuvenharrison

**Comment by reuvenharrison** [comment]
Hi @WillGunn,
This functionality is currently not planned because it requires a major effort to implement and I haven't seen much demand for it.
If you think otherwise, please let me know.
Regards,
Reuven

**Comment by reuvenharrison** [comment]
Update:
We are currently in the process of adding a change-log feature that may answer your requirement.
Please see more details here: https://github.com/Tufin/oasdiff/discussions/229
